TrainingExtensions/tensorflow/src/python/aimet_tensorflow/quantizer_info.py

```diff
@@ -360,11 +440,22 @@ def __setstate__(self, state):
         # Create the cpp tensor quantizer reference
         self.quant_op_name = state.quant_op_name
         self.quantizer_type = state.quantizer_type
-        self.tensor_quantizer = libpymo.TensorQuantizer(state.quant_scheme,
-                                                        state.rounding_mode)
-        self.tensor_quantizer.setStrictSymmetric(state.use_strict_symmetric)
-        self.tensor_quantizer.setUnsignedSymmetric(state.use_unsigned_symmetric)
-        self.tensor_quantizer.isEncodingValid = state.is_encoding_valid
+        # If per channel quantization is enabled for a parameter
+        if state.num_channels > 0:
+            self.tensor_quantizer = []
+            for _ in range(state.num_channels):
+                tensor_quantizer = libpymo.TensorQuantizer(state.quant_scheme,
+                                                           state.rounding_mode)
+                tensor_quantizer.setStrictSymmetric(state.use_strict_symmetric)
+                tensor_quantizer.setUnsignedSymmetric(state.use_unsigned_symmetric)
+                tensor_quantizer.isEncodingValid = state.is_encoding_valid
+                self.tensor_quantizer.append(tensor_quantizer)
+        else:
+            self.tensor_quantizer = libpymo.TensorQuantizer(state.quant_scheme,
+                                                            state.rounding_mode)
+            self.tensor_quantizer.setStrictSymmetric(state.use_strict_symmetric)
+            self.tensor_quantizer.setUnsignedSymmetric(state.use_unsigned_symmetric)
+            self.tensor_quantizer.isEncodingValid = state.is_encoding_valid
```

Comment:
please encapsulate repetitive code in a function, but it can be done as part of next pr, if thats more manageable